mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -336,13 +354,24 @@ func InvokeGenerationFromSheet(wg *sync.WaitGroup) error {
 	}
 	wg.Wait()
 	close(spreadsheeetChan)
+	utils.ProcessRelationships(relationshipCSVHelper)
 	wgForSpreadsheetUpdate.Wait()
 	return nil
 }
 
 // For registrants eg: meshery, whose components needs to be directly created by referencing meshery/schemas repo.
 // the sourceURL contains the path of models component definitions
 func GenerateDefsForCoreRegistrant(model utils.ModelCSV, ComponentCSVHelper *utils.ComponentCSVHelper) error {
+	registrant := "meshery"
+	if models, exists := ComponentCSVHelper.Components[registrant]; exists {
```

Comment:
```suggestion
	if models, exists := ComponentCSVHelper.Components[registrant][model.Model]; exists {
```

And remove the if branch.

Reply:
Okay